Sources/MapboxMaps/Foundation/Extensions/Turf/Geometry.swift

```diff
@@ -50,8 +50,8 @@ extension MapboxCommon.Geometry {
 
     /// Allows a `Geometry` object to be initialized with a `Geometry` object.
     /// - Parameter geometry: The `Geometry` object to transform into the `Geometry` type.
```

Comment:
I think the comment above needs to be updated too.

Reply:
Updated.